macos/docs/Controls/Button.md

```diff
@@ -0,0 +1,36 @@
+# Button
+
+## Overview
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+![ButtonViews.png](.attachments/ButtonViews.png)
+
+## Best Practices
+### Do
+- Make sure the label conveys a clear purpose of the button to the user.
+- Describe the action the button will perform, ideally with a verb. Use concise, specific, self-explanatory labels, usually a single word.
+
+### Don't
```

Comment:
Should clients not override our cell? Or can/should they?

Reply:
... That's effectively undefined behavior. Is it worth mentioninig?

Reply:
Per another comment in this review, I find it nice when frameworks callout what behavior would be undefined :). Particularly when it's something like tampering with a cell which is something anyone who has ever played with an NSButton has done

Reply:
Added a line under "Usage" in the documentation, since that's the most "technical" part of the documentation